src/analysis/retail/shaman/enhancement/modules/talents/Ascendance.tsx

```diff
@@ -478,6 +509,23 @@ class Ascendance extends MajorCooldown<AscendanceCooldownCast> {
       );
     }
   }
+
+  get guideSubsection() {
+    return (
+      this.active && (
+        <>
+          <CooldownUsage
+            analyzer={this}
+            title={
+              this.selectedCombatant.hasTalent(TALENTS_SHAMAN.ASCENDANCE_ENHANCEMENT_TALENT)
+                ? 'Ascendance'
+                : 'Deeply Rooted Elements'
```

Comment:
Consider adding `SpellLink` for these spell names

Reply:
Done.